gcp-serverless/package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "test": "jest",
     "standard": "standard",
-    "cs-fix": "standard --fix"
-
+    "cs-fix": "standard --fix",
+    "local-check": "node -e 'require(\"./index\").makeRequest(process.env.PARTICLE_DEVICE_ID)'"
```

Comment:
calling `node` as an executable from within an npm script seems... odd. I see what you're doing, and it seems like there must be a better way. But I'm not sure what it is.

Reply:
Yeah, I'm sure there's a better way, but the ergonomics, `npm run local-check` would be the same. I expect I'll come across something cleaner, but I don't think it is necessary to spend time hunting for that now.

Reply:
Fair enough